src/com/dakl/Main.java

```diff
@@ -194,7 +200,9 @@ public static Space parseSpace(String piece_str) {
     }
 }
 
+// REVIEW : separate files for classes
```

Comment:
As a general practice, it is always best to separate the individual classes (e.g. CheckerBoard) into their own files (and possibly even a different package)